src/Client/PredisClient.php

```diff
@@ -230,8 +230,11 @@ public function count(string $prefixKey, array $criterias = []): int
         foreach ($criterias as $property => $value) {
             $arguments[] = "@$property:$value";
         }
+        if ($criterias === []) {
+            $arguments[] = '*';
+        }
 
-        $rawResult = call_user_func_array([$this->redis, 'executeRaw'], $arguments);
+        $rawResult = call_user_func_array([$this->redis, 'executeRaw'], [$arguments]);
```

Comment:
why add arguments in an array ?

Reply:
because call_user_func_array calls &$errors